fastly-rails.gemspec

```diff
@@ -14,9 +14,11 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.rdoc"]
   s.test_files = Dir["test/**/*"]
 
-  s.add_dependency "rails"
+  s.add_dependency "railties"
   s.add_dependency 'fastly', '~> 1.2.1'
 
+  s.add_runtime_dependency('mime-types', ['>= 1.16', '< 3'])
```

Comment:
It seems a bit arbitrary, yes. I've copied it from [this PR](https://github.com/cucumber/cucumber-rails/pull/294/files#diff-5bacb7ff9bc30c91a517ad25b85d4498R19) on the `cucumber-rails` project. Why it seemed fine for me was:
- `mime-types` v1.16 is the last available tagged version on their Github project page (https://github.com/mime-types/ruby-mime-types/tags?after=v1.22)
- `mime-types` v3 and upwards only support Ruby 2.x